_15thnight/api/account.py

```diff
@@ -66,6 +61,7 @@ def forgot_password():
     """
     Send a password reset email.
     """
+    reset_token_life = current_app.config.get("RESET_TOKEN_LIFE", "")
```

Comment:
Should these really need a default value? They should have a value set for this. If not maybe they should crash? Or maybe we should import configdist and pass that as the default value for this pattern.